Updated portal messaging to show email

troyciesco opened the pull request:
ref https://linear.app/ghost/issue/PROD-2620

- After a user signs in, if OTC is turned on, the next screen ("check your email") will now display the email address instead of just "your inbox"
- This will make it easier to see if there was a typo in the email address, in the event an email never arrives
- different messages for sign in without OTC available, and sign in with OTC available

<img width="485" height="225" alt="Screenshot 2025-09-23 at 12 36 45 PM" src="https://github.com/user-attachments/assets/b1909511-fc8b-4141-96ae-672a59eeea83" />

sign in without OTC is _not_ impacted:
<img width="200" height="204" alt="Screenshot 2025-09-23 at 12 39 53 PM" src="https://github.com/user-attachments/assets/0c8fccda-980e-41c5-9975-9feba14f1829" />

nor is sign up:
<img width="200" height="164" alt="Screenshot 2025-09-23 at 12 40 19 PM" src="https://github.com/user-attachments/assets/9f792a4c-1bd7-46ea-afb4-5a7dc0d25f0a" />

troyciesco committed:
Updated portal messaging to show email

troyciesco committed:
update messaging behind labs flag

troyciesco committed:
fix translations

troyciesco committed:
semantic tweaks

troyciesco committed:
remove signup

kevinansfield reviewed (changes requested):
As discussed in our sync call, it would be good to put the updated strings behind a labs flag so we have new strings ready for early translation rather than production strings being cleared out immediately.

troyciesco committed:
better conditional email in description logic, add labs flag

troyciesco committed:
fix tests

troyciesco commented on ghost/core/test/unit/frontend/helpers/__snapshots__/ghost_head.test.js.snap:
```diff
@@ -1097,57 +1097,6 @@ Object {
 }
 `;
 
-exports[`{{ghost_head}} helper includes tinybird tracker script when config is set does not include tracker script when neither trafficAnalytics nor trafficAnalyticsTracking is set 1 1`] = `
```
hmm more changes than expected here 👀

troyciesco replied:
ok reviewed a bit, these few analytics snapshots can be cleaned up safely bc the tests no longer exist. In particular, trafficAnalyticsTracking is an old flag that doesn't exist anymore.

troyciesco committed:
update jsdoc

troyciesco commented:
> As discussed in our sync call, it would be good to put the updated strings behind a labs flag so we have new strings ready for early translation rather than production strings being cleared out immediately.

@kevinansfield I'd originally used the otc flag to gate this change, thinking that we'd want to switch to email text at the same time otc is GA, but now realize it doesn't necessarily make sense to do that, especially since a few otc translations have already been done.

I've updated this PR with a new flag specifically for the email text, and cleaned up the creation of the `popupDescription` variable a bit so there wasn't so much hard-to-parse if/else logic.

troyciesco committed:
remove extraneous feature flag

troyciesco committed:
update translations

troyciesco committed:
remove unneeded test
